### DISCUSSION TITLE
Problems updating to v5

### DISCUSSION THREAD

**Discussion by krichel89** [LycheeOrg/Lychee#2133] [Q&A] [answered]
Trying to update to v5 the manual way. v4.11 worked fine.

The database was updated, but I cannot log in anymore. The login button has no function, links to public images are broken.
I pointed the APP_URL=https://***.de/Lychee/public/app 
Did I miss something?

**Comment by d7415** [comment, marked as answer]
Subdirectory hosting is not currently supported (but is planned) in v5. Setting `LIVEWIRE_ENABLED=false` in your `.env` should work around it.

Additionally, `APP_URL` should be the URL you access Lychee from, in your case presumably `APP_URL=https://***.de/Lychee`.

Just in case I will state that the root (above the `public` folder) should **not** be publicly accessible.

**Reply by krichel89** [reply]
Thanks for your fast response, that was very helpful.
Only the APP_URL had to be pointed to the public folder, in my example `APP_URL=https://***.de/Lychee/public`

**Reply by d7415** [reply]
One more time:

> Just in case I will state that the root (above the `public` folder) should **not** be publicly accessible.

If you are accessing your gallery from `https://***.de/Lychee/public` then you are probably running a misconfigured and very insecure setup.
